apps/explorer/src/state/network/NetworkUpdater.tsx

```diff
@@ -6,13 +6,15 @@ import { setNetwork } from './actions'
 import { updateWeb3Provider } from '../../api/web3'
 import { web3 } from '../../explorer/api'
 import { SupportedChainId } from '@cowprotocol/cow-sdk'
-import { CHAIN_ID_TO_URL_PREFIX, NETWORK_PREFIXES } from '../../consts/network'
+import { CHAIN_INFO } from '@cowprotocol/common-const'
+import { NETWORKS_PREFIXES } from './const'
 
-const MAINNET_PREFIX = CHAIN_ID_TO_URL_PREFIX[SupportedChainId.MAINNET]
-const NETWORK_PREFIXES_RAW: [SupportedChainId, string][] = Object.keys(CHAIN_ID_TO_URL_PREFIX).map((key) => [
-  key as unknown as SupportedChainId,
-  CHAIN_ID_TO_URL_PREFIX[key],
-])
+const MAINNET_PREFIX = CHAIN_INFO[SupportedChainId.MAINNET].urlAlias
+const NETWORK_PREFIXES_RAW: [SupportedChainId, string][] = Object.keys(CHAIN_INFO).map((key) => {
+  const chainId = +key as unknown as SupportedChainId
```

Comment:
No need for the cast after converting to number.
See this fix https://github.com/cowprotocol/cowswap/blob/784ce32bcfa7908999f3130357582602f02802a5/apps/explorer/src/state/network/NetworkUpdater.tsx#L13

```suggestion
  const chainId = +key
```